www/searchutil.php

```diff
@@ -974,6 +974,16 @@ function doSearch($db, $term, $searchType, $sortby, $limit, $browse)
                     . "if($likeExpr,0,1),"
                     . "relevance desc,";
 
+    // in game searches, implicitly match by TUID and IFID
+    if ($searchType == "game" && count($words) == 1 && count($extraJoins) == 0) {
+        $extraJoins['/ifid/'] = true;
+        $tableList .= " left outer join ifids "
+                            . "on games.id = ifids.gameid";
+
+        $txt = mysql_real_escape_string($words[0], $db);
```

Comment:
I think real_escape_string shouldn't be avoided, now that there's support for bind parameters.
This is what I did earlier:
```php
$whereParameters = [];
...
$whereParameters[] = $words[0];
$whereParameters[] = $words[0];
...
$bindParamters = array_merge($tagsToMatch, $whereParameters);
...
$result = mysqli_execute_query($db, $sql, $bindParamters);
```